content/_data/changelogs/lts.yml

```diff
@@ -2837,6 +2837,143 @@
+      message: |-
+        Remove the ability to download update center metadata using the user's browser (deprecated since 2015).
+        Jenkins will no longer inform about available updates without a connection to update sites.
+        We recommend the use of a local mirror of our update sites, or a self-hosted update center like <a href="https://github.com/jenkinsci/juseppe">Juseppe</a> in these situations.
+    - type: rfe
+      pull: 4242
+      authors:
+        - daniel-beck
+      message: |-
+        Stop bundling Maven Plugin and Subversion Plugin with Jenkins.
+        In very rare cases, this could result in problems when attempting to install plugins compatible with Jenkins before 1.310.
+        The Jenkins project is currently not publishing any such plugins.
+    - type: rfe
+      message: |-
+        Logging UI: Reorder sidepanel entries, add a note that "all log messages" will only include entries on level INFO and up.
+      pull: 4305
+      authors:
+        - daniel-beck
+    - type: rfe
+      message: |-
+        Update the Plugin Manager Updates tab with more information about incompatible dependencies.
+      pull: 4299
+      authors:
+        - andipabst
+    - type: rfe
+      issue: 17459
+      message: |-
+        Build status balls on the build trend page now link to the respective build's console output.
+      pull: 4312
+      authors:
+        - rcsalome
+    - type: bug
+      pull: 4246
+      references:
+        - issue: 58296
+        - issue: 53462
+        - url: https://bugzilla.mozilla.org/show_bug.cgi?id=1370630
+          title: Firefox issue 1370630
+      authors:
+        - thomasgl-orange
+      message: |-
+        Revert changes in forms submission in Jenkins classic UI with Firefox have caused a regression on forms with "file" inputs.
+        These were made to anticipate a bugfix in Firefox which has been backed out since.
+        (regression in 2.173)
+    - type: rfe
+      pull: 4209
+      references:
+        - issue: 19760
+        - issue: 31209
+      authors:
+        - slonopotamus
+      message: |-
+        Remove 100 character length limitation of build description in build history widget.
+    - type: rfe
+      pull: 4341
+      message: |-
+        Increase the number of datapoints recorded for multistage time series graphs such as those used for load statistics.
+      authors:
+        - daniel-beck
+    - type: rfe
+      pull: 4330
+      authors:
+        - tsparber
+      message: |-
+        Internal: Remove unused <code>jenkins-slave.xml</code> file template from core.
+        Requires WMI Windows Agents plugin version 1.3.1 (Released Mar 14 2017) or newer.
+    - type: rfe
+      pull: 4340
+      references:
+        - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-3.36
+          title: Remoting release notes
+      message: |-
+        Update Remoting to 3.36 to add new command line options "-help" and "-version".
+      authors:
```

Comment:
There were more updates between versions, and explicit versions here might be confusing

Reply:
We've always done these this way.